homeassistant/components/fritz/switch.py

```diff
@@ -245,7 +245,10 @@ def wifi_entities_list(
 ) -> list[FritzBoxWifiSwitch]:
     """Get list of wifi entities."""
     _LOGGER.debug("Setting up %s switches", SWITCH_TYPE_WIFINETWORK)
-    std_table = {"ax": "Wifi6", "ac": "5Ghz", "n": "2.4Ghz"}
+    if fritzbox_tools.model == "FRITZ!Box 7390":
+        std_table = {"n": "5Ghz"}
+    else:
+        std_table = {"ax": "Wifi6", "ac": "5Ghz", "n": "2.4Ghz"}
```

Comment:
More a style thing... but `else` can almost always be avoided :)

```suggestion
    std_table = {"ax": "Wifi6", "ac": "5Ghz", "n": "2.4Ghz"}
    if fritzbox_tools.model == "FRITZ!Box 7390":
        std_table = {"n": "5Ghz"}
```

Reply:
Oh yes, easy change. I just though it was more readable ;-)

Reply:
What else? :D

Reply:
:-D